tests/ops/test_identity.py

```diff
@@ -15,11 +15,36 @@
 import numpy as np
 import pytest
 
+import pennylane as qml
 from pennylane import Identity
 
 
 @pytest.mark.parametrize("wires", [[0], [0, 1], ["a", "b", "c"], [100, "xasd", 12]])
 class TestIdentity:
+
+    # pylint: disable=protected-access
+    def test_flatten_unflatten(self, wires):
+        """Test the flatten and unflatten methods of identity."""
```

Comment:
Would it make sense to copy/paste the couple of pytree-related tests from `test_parametric_ops.py` to `test_non_parametric_ops.py` so we cover them all too?

Reply:
added Identity to the list of `NON_PARAMETRIZED_OPERATIONS` in `test_parametric_ops.py`.  Still kinda weird that we have a bunch of non-parametric ops tested in `test_parametric_ops.py`, but I don't feel like actually fixing that weirdness.

Reply:
😂 alright

Reply:
reminds me of this classic python thing:
```pycon
>>> qml.ops.qubit.parametric_ops_multi_qubit.Hadamard
<class 'pennylane.ops.qubit.non_parametric_ops.Hadamard'>
```
the multi-qubit, parametric Hadamard operator.